Makefile

```diff
@@ -48,3 +49,9 @@ setup:
 .PHONY: test
 test: dist/ramda.js
 	npm test
+	find src -name '*.js' -not -path 'src/internal/*' \
+	| sed 's:src/\(.*\)[.]js:exports.\1 = require("./src/\1");:' >index.js
+	sed -i '' '/"main":/d' package.json
+	$(MOCHA) -- $(shell find test -name '*.js' -not -name 'installTo.js' -not -name 'test.examplesRunner.js')
+	git checkout -- package.json
```

Comment:
Revert our change to __package.json__. This will also revert any previous unstaged changes.